packages/valory/services/market_maker/service.yaml

```diff
@@ -71,6 +71,8 @@ models:
       multisend_batch_size: ${MULTISEND_BATCH_SIZE:int:1}
       ipfs_address: ${IPFS_ADDRESS:str:https://gateway.autonolas.tech/ipfs/}
       mech_contract_address: ${MECH_CONTRACT_ADDRESS:str:0x77af31de935740567cf4ff1986d04b2c964a786a}
+      answer_retry_intervals: ${ANSWER_RETRY_INTERVALS:list:[0, 86400, 259200, 604800,
+        1209600]}
```

Comment:
Wouldn't it be simpler if you implemented a backoff logic and made the backoff factor configurable?